tests/test_server.py

```diff
@@ -104,13 +105,17 @@ def test_version(rasa_app: SanicTestClient):
 
 def test_status(rasa_app: SanicTestClient):
     _, response = rasa_app.get("/status")
+    model_file = response.json["model_file"]
+    test_folder_pattern = re.search(r"models\d/\d+-\d+.tar.gz", model_file)
     assert response.status == 200
     assert "fingerprint" in response.json
     assert "model_file" in response.json
+    assert "models/" in model_file or test_folder_pattern
```

Comment:
can you not test `os.path.isfile`? What problem did you run into so that you had to use the regex ?

Reply:
I just forgot about it honestly so I updated it back to check for that.